src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -208,6 +219,7 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
         fieldName: 'isAdmin',
         minWidth: (tokenPresent) ? 150 : 100,
         maxWidth: (tokenPresent) ? 150 : 100,
+        ariaLabel: messages['Administrator permission']
```

Comment:
The new function to render the fallback text when localisation has not been added yet is going to be useful here